static/js/copyPasteEvents.js

```diff
@@ -1,3 +1,5 @@
+var _ = require("underscore");
```

Comment:
This is client-side code, so there's no node_modules. I'm actually not sure how require() operates on the client side.